libs/stats/tests/test_utils.py

```diff
@@ -117,3 +124,180 @@ def test_bin_seasonal_mk_season_rules(months, anchor):
     assert {'01--P6M', '07--P6M'} == set(season_rules.values())
 
 
+@pytest.fixture
+def fc_definition():
+    definition = {
+        'name': 'ga_ls_fc_3',
+        'metadata': {
+            'product': {'name': 'ga_ls_fc_3'},
+            'properties': {'odc:file_format': 'GeoTIFF', 'odc:product_family': 'fc'}
+        },
+        'description': 'Geoscience Australia Landsat Fractional Cover Collection 3',
+        'measurements': [
+            {'name': 'bs', 'dtype': 'uint8', 'units': 'percent', 'nodata': 255, 'aliases': ['bare']},
+            {'name': 'pv', 'dtype': 'uint8', 'units': 'percent', 'nodata': 255, 'aliases': ['green_veg']}, 
+            {'name': 'npv', 'dtype': 'uint8', 'units': 'percent', 'nodata': 255, 'aliases': ['dead_veg']},
+            {'name': 'ue', 'dtype': 'uint8', 'units': '1', 'nodata': 255, 'aliases': ['err']}],
+        'metadata_type': 'eo3'
+    }
+    return definition
+
+
+@pytest.fixture
+def wo_definition():
+    definition = {
+        'name': 'ga_ls_wo_3',
+        'metadata': {
+            'product': {'name': 'ga_ls_wo_3'},
+            'properties': {'odc:file_format': 'GeoTIFF', 'odc:product_family': 'wo'}
+        },
+        'description': 'Geoscience Australia Landsat Water Observations Collection 3',
+        'measurements': [{'name': 'water', 'dtype': 'uint8', 'units': '1', 'nodata': 1}],
+        'metadata_type': 'eo3'
+    }
+    return definition
+
+
+@pytest.fixture
+def dc():
+    return Datacube()
```

Comment:
So tests are failing because database is not initialized. How about we get rid of the dependency on the database for this test suite. For this to work I suggest we extract this code into 

https://github.com/opendatacube/odc-tools/blob/4c171c4bda580b8ff3a243a4079a716f0563d3f9/libs/index/odc/index/_index.py#L366-L378

into `str -> MetadataType` function that constructs and returns `MetadataType` object from a standard name.